jasmine_overrides.rb

```diff
@@ -0,0 +1,77 @@
+class StaticOrHaml
```

Comment:
Would not be better to put this somewhere under `lib`?

Reply:
Maybe, how does that work?

Is it safe to move to lib if this should not be loaded by the app, but only by the Rakefile?

Reply:
Well, I'm not 100% sure, but you could set/check for `RAILS_ENV`, right?

Reply:
So far, it seems to work :) Added ..

Reply:
Well, the `lib` is managed by autoload, and you're not loading that class anywhere, but magic can happen, so I'd rather wrap an ENV-checking around it anyway.

Reply:
Ok, in that case, undoing the last commit, doesn't sound like this can live in `lib/` then ;).

There is no ENV difference betweeen `rake environment jasmine` and `rails s`.

Reply:
@himdel I think it should be :wink: just for consistency with rspec and other tests. Or is there any other kind of ENV var specific to jasmine?

Reply:
Care to elaborate please?

What do you expect me to do here? There is nothing related to rspec in lib.

Reply:
> Or is there any other kind of ENV var specific to jasmine?

Nope, no RAILS_ENV, and nothing mentioning jasmine in the rest of ENV.

Reply:
Okay, I'll find a solution, just gimme some time

Reply:
I was thinking a little, I think this could live inside the `config` folder, as far as I see the jest setup is there as well. So can you please move it there until I come up with something better?

Reply:
<3 Thanks, moved to `config/` :)